packages/react-data-grid/src/common/types.ts

```diff
@@ -168,11 +161,13 @@ export type CellContentRendererProps<TRow> = Pick<CellRendererProps<TRow>,
 | 'rowIdx'
 | 'rowData'
 | 'column'
-| 'cellMetaData'
 | 'expandableOptions'
 | 'isRowSelected'
 | 'onRowSelectionChange'
 | 'isSummaryRow'
+| 'onDeleteSubRow'
+| 'onCellExpand'
+| 'getCellActions'
```

Comment:
I am not sure why we added these props but SubRows and CellActions can be handled via a custom formatter